client-misk-dynamodb/src/test/kotlin/app/cash/backfila/client/misk/dynamodb/DynamoDbLastEvaluatedKeyTest.kt

```diff
@@ -30,10 +30,11 @@ class DynamoDbLastEvaluatedKeyTest {
   fun `one big segment with one second batch execution time works`() {
     testData.addThriller()
 
-    // TODO(mikepaw) use batch size to change how many items show up in a batch.
+    // Pause so at most two dynamo batches run per runBatch service call
     val run = backfila.createWetRun<PausingBackfill>(
-        parameters = PausingBackfill.Parameters(1, 1, 128L)
+        parameters = PausingBackfill.Parameters(1, 1, 500L, 5)
```

Comment:
can you use named parameters to make this more readable?

Reply:
For sure. Done.